conf/cassandra/migration/default/V2__query_alerts.cql

```diff
@@ -0,0 +1,56 @@
+-- description: creates initial schema
+-- authoredAt: 1370028268000
+-- up:
+
+-- stage: 1
+DROP MATERIALIZED VIEW alerts_by_cluster;
```

Comment:
Or at least these seem redundant as they are also in the v1 file.

Reply:
Downs must match the ups.